src/navigator/offlineNavigator.js

```diff
@@ -31,7 +31,8 @@ import testContextBuilder from 'taoQtiTest/runner/helpers/testContextBuilder';
 export default function offlineNavigatorFactory(itemStore, responseStore) {
     var testContext,
         testMap,
-        offlineJumpTableHelper = offlineJumpTableFactory(itemStore, responseStore);
+        offlineJumpTableHelper = offlineJumpTableFactory(itemStore, responseStore),
+        itemStore = itemStore;
```

Comment:
What is the purpose of this variable? it is already given as a parameter.
There are two issues:
- the variable is redeclared with the same name
- the new variable is not assigned later, so keeping the former one should be enough

Reply:
fixed